receiver/jmxreceiver/config.go

```diff
@@ -60,10 +57,13 @@ type Config struct {
 	RemoteProfile string `mapstructure:"remote_profile"`
 	// The SASL/DIGEST-MD5 realm
 	Realm string `mapstructure:"realm"`
-	// Map of property names to values to pass as system properties when running JMX Metric Gatherer
-	Properties map[string]string `mapstructure:"properties"`
 	// Array of additional JARs to be added to the the class path when launching the JMX Metric Gatherer JAR
 	AdditionalJars []string `mapstructure:"additional_jars"`
+	// Map of resource attributees used by the Java SDK Autoconfigure to set resource attributes
+	ResourceAttributes map[string]string `mapstructure:"resource_attributes"`
+	// Log level used by the JMX metric gatherer. Should be one of:
+	// `"trace"`, `"debug"`, `"info"`, `"warn"`, `"error"`, `"off"`
+	LogLevel string `mapstructure:"log_level"`
```

Comment:
Any thoughts on whether this should re-use the log level set in the collector's `service.logging` setting instead of adding its own?

Reply:
I'm open to it. The only drawbacks I see are that there's no Trace equivalent in [zap.Level](https://github.com/uber-go/zap/blob/master/zapcore/level.go#L138) and several levels (fatal, panic, dpanic, error) would probably all map to error (or possibly panic/dpanic to "off").

Reply:
@codeboten implemented it so if no user config is provided, it attempts to translate from zap logger level, otherwise it uses the user provided config. Let me know what you think!